python/paddle/trainer_config_helpers/layers.py

```diff
@@ -3334,6 +3335,60 @@ def block_expand_layer(input,
 
 
 @wrap_name_default()
+@layer_support()
+def maxout_layer(input,
+                 channels,
+                 groups,
+                 size_x=None,
+                 size_y=None,
+                 name=None,
+                 layer_attr=None):
+    """
+    A layer to do max out on conv layer output.
+      - Input: output of a conv layer.
+      - Output: feature map size same as input. Channel is (input channel) / groups.
+
```

Comment:
would be better if we add the reference paper in annotation. 
http://www.jmlr.org/proceedings/papers/v28/goodfellow13.pdf
https://arxiv.org/pdf/1312.6082v4.pdf